js/logging.js

```diff
@@ -2,12 +2,10 @@
 
 /* eslint strict: ['error', 'never'] */
 
-/* global $: false */
-/* global textsecure: false */
-
 const electron = require('electron');
 const bunyan = require('bunyan');
 const _ = require('lodash');
+const superagent = require('superagent');
```

Comment:
The [`fetch` API is quite low-level](https://medium.com/@shahata/why-i-wont-be-using-fetch-api-in-my-apps-6900e6c6fe78) so I added `superagent` (by reputable authors) as a more high-level alternative that supports timeouts, form submission, and is also universal (isomorphic).

Reply:
Hm. We may have to think a little bit more about whether `superagent` is the library we want to replace `fetch` with. One thing I do know is that, given that this will need to be ported to the Chrome app, we'll have to write it in jQuery either way.

Reply:
As discussed offline, I’ll change this to use `got`. As for Chrome, I will deal with that separately rather than having that app influence our code style / libraries for new changes.